openshift-examples/keycloak-https.json

```diff
@@ -233,14 +263,30 @@
                                         "name": "DB_VENDOR",
                                         "value": "${DB_VENDOR}"
                                     },
+                                    {
+                                        "name": "DB_ADDR",
+                                        "value": "${DB_ADDR}.${NAMESPACE}.svc.cluster.local"
```

Comment:
You should add .${NAMESPACE}.svc.cluster.local here as that will only work in some situations. You have effectively broken support for connecting to an external DB by doing this.

Reply:
sorry, I do not understand what is wrong with it. What would be the correct value?